### ISSUE TITLE
Cannot install the Mac version

### ISSUE THREAD

**Issue by henry-bao** [Devsfy/wiregui#32]
An error appeared and said `“wiregui” is damaged and can’t be opened. You should move it to the Trash."` After I tried to open the application

<img width="524" alt="Screenshot 2023-02-21 at 5 24 48 AM" src="https://user-images.githubusercontent.com/62913901/220356885-204242c0-03c0-43d3-81bb-37a2f1512ddc.png">

**Comment by wuzi** [comment]
indeed, I think some new version from Mac broke it, will need to rebuild it
I don't have a Mac with the development tools installed to build it at the moment, once I have I can try to rebuild

**Comment by Kr4is** [comment]
Hello! I have a similar issue, i had the following message: `You can’t open the application “wiregui” because this application is not supported on this Mac.`

Could be possible generate the version for intel amd64 based architectures?

Thanks in advance!

**Comment by henri26mobu** [comment]
Hi! For reference, macOS support (Intel x64 & Apple Silicon arm64) is available in this active fork: https://github.com/henri26mobu/wiregui
